oraichain/assetlist.json

```diff
@@ -29,6 +29,406 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg"
         }
       ]
+    },
+    {
+      "description": "Atom token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
+          "exponent": 0
+        },
+        {
+          "denom": "ATOM",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
```

Comment:
If there's a base denom with an ibc hash, then this was likely ibc transferred. Where did it come from? Please use full trace paths for all tokens.